firmware/thruna/PDM/Src/App/App_powerManager.c

```diff
@@ -0,0 +1,46 @@
+#include "App_PowerManager.h"
+
+typedef struct
+{
+    bool efuses[NUM_EFUSE_CHANNELS];
+} PowerStateConfig;
+
+static const PowerStateConfig power_states_config[NUM_POWER_STATES] = {
+    [POWER_MANAGER_CONTRACTOR_OPEN] = {
+        .efuses = {
+            [EFUSE_CHANNEL_AIR] = false,
+            [EFUSE_CHANNEL_LVPWR] = false,
+            [EFUSE_CHANNEL_EMETER] = false,
+            [EFUSE_CHANNEL_AUX] = false,
+            [EFUSE_CHANNEL_DRS] = false,
+            [EFUSE_CHANNEL_FAN] = false,
+            [EFUSE_CHANNEL_DI_LHS] = false,
+            [EFUSE_CHANNEL_DI_RHS] = false,
+        }
+    },
+    [POWER_MANAGER_CONTRACTOR_CLOSED] = {
+        .efuses = {
+            [EFUSE_CHANNEL_AIR] = true,
+            [EFUSE_CHANNEL_LVPWR] = true,
+            [EFUSE_CHANNEL_EMETER] = true,
+            [EFUSE_CHANNEL_AUX] = true,
+            [EFUSE_CHANNEL_DRS] = true,
+            [EFUSE_CHANNEL_FAN] = true,
+            [EFUSE_CHANNEL_DI_LHS] = true,
+            [EFUSE_CHANNEL_DI_RHS] = true,
+        }
+    }
+};
+
+void PowerManager_Init()
```

Comment:
Make sure you call this init function somewhere in `main.c` for it to actually get run.